apps/marginfi-v2-ui/src/pages/index.tsx

```diff
@@ -10,11 +10,26 @@ import { OverlaySpinner } from "~/components/desktop/OverlaySpinner";
 import { useConnection } from "@solana/wallet-adapter-react";
 import { Desktop, Mobile } from "~/mediaQueries";
 
-const AccountSummary = dynamic(async () => (await import("~/components/desktop/AccountSummary")).AccountSummary, {
+const DesktopAccountSummary = dynamic(
+  async () => (await import("~/components/desktop/DesktopAccountSummary")).DesktopAccountSummary,
+  {
+    ssr: false,
+  }
+);
+const MobileAccountSummary = dynamic(
+  async () => (await import("~/components/mobile/MobileAccountSummary")).MobileAccountSummary,
+  {
+    ssr: false,
+  }
+);
+const AssetsList = dynamic(async () => (await import("~/components/desktop/AssetsList")).AssetsList, { ssr: false });
+const MobileAssetsList = dynamic(async () => (await import("~/components/mobile/MobileAssetsList")).MobileAssetsList, {
+  ssr: false,
+});
+
+const UserPositions = dynamic(async () => (await import("~/components/desktop/UserPositions")).UserPositions, {
   ssr: false,
 });
```

Comment:
best would be to have the dynamic imports done within their respective breakpoint to avoid loading both every time